snekbox/api/resources/eval.py

```diff
@@ -54,6 +61,29 @@ class EvalResource:
     def __init__(self, nsjail: NsJail):
         self.nsjail = nsjail
 
+    @validate(
+        resp_schema={
+            "versions": {"type": "array", "items": {"type": "str"}},
+        }
+    )
+    def on_get(self, _: falcon.Request, resp: falcon.Response) -> None:
+        """
+        Get information about the server.
+
+        Response format:
+        >>> {
+        ...     "versions": ["Python 3.9", "Python 3.10", "Python 3.12 Beta 1"]
```

Comment:
Maybe we should follow a naming convention more like pyenv, which distinguishes between cpython, pypy, etc.

Reply:
That sounds smart, should be fairly easy to modify in the json file.

Reply:
f564fd70